integrations/acquisition/covidcast/test_covidcast_meta_caching.py

```diff
@@ -67,11 +67,11 @@ def test_caching(self):
 
     # insert dummy data
     self.cur.execute(f'''
-      INSERT INTO
-        `covidcast` (`id`, `source`, `signal`, `time_type`, 
-        `geo_type`, `time_value`, `geo_value`, `value_updated_timestamp`, 
-        `value`, `stderr`, `sample_size`, `direction_updated_timestamp`, 
-        `direction`, `issue`, `lag`, `is_latest_issue`, `is_wip`) 
+      INSERT INTO covidcast
+          (id, data_source, signal, time_type, geo_type,
+          time_values, geo_value, value_updated_timestamp,
+          value, stderr, sample_size, direction_updated_timestamp, direction, issue,
+          lag, result, is_wip)
```

Comment:
Keep the backticks! Some of the column names (notably `signal`) are reserved words, and the query won't run if you don't escape the column names properly.